index.js

```diff
@@ -14,4 +13,29 @@ app.get('/', (req, res) => res.send("Hello"));
 function onListen() {
     console.log(`Listening on port ${port}`);
 }
-app.listen(port, onListen);
\ No newline at end of file
+app.listen(port, onListen);
+
+// connect to heroku database
+const { Client } = require('pg');
```

Comment:
Definitely fine for now but FYI there's a [pooling ](https://node-postgres.com/features/pooling)option that might be good to use to reduce latency from only having a single client connection from the server (e.g. if we have three client requests we'll have to serialize them). I think we're only allowed 20 concurrent connections with the free postgres from Heroku tho so  that probably wouldn't make that much difference.